app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,90 @@
+<div class="row inset">
+  <section class="col col-720">
+<p>Teaching computing you'll help pupils become responsible and confident users of information and technology.</p>
+<p>In an increasingly digital world, you could inspire pupils to look at a future role in technology. With roles in technology making up 14% of all job opportunities in the UK.</p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee computing teachers</a>.</p>
+</section>
+  <section class="col col-space-s col-content-max">
+<%= render TextBlockWithTwoImagesComponent.new(
+heading: "Help your pupils learn new skills",
+colour: "pink",
+image_1_path: "static/images/content/hero-images/0035.jpg",
+image_1_alt: "",
+image_2_path: "static/images/content/hero-images/0037.jpg",
+image_2_alt: "",
+) do %>
+<p>Computing focuses on 3 areas:</p>
+<ul>
+<li>information and communication technology</li>
+<li>digital literacy</li>
+<li>computer science</li>
+</ul>
+<p>You'll inspire pupils to access new fields and teach them new, transferable skills that will open lots of sectors, from engineering to healthcare. Leading them to future careers like software engineering and web design.</p>
+<% end %>
+</section>
+<section class="col col-720">
+<h2>What you'll be teaching</h2>
+<p>At GCSE and A level you'll teach computer science. You will teach themes from the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum</a>, with opportunities to develop your pupils' computational thinking.  </p>
```

Comment:
@caitlin-reidy **You will** you'll

Reply:
@caitlin-reidy do we need to say (opens in new window) as we're taking users to GOV.UK

Reply:
This is a wider issue that we need to resolve across the site (we have a ticket on the backlog) - and I would suggest we do this programmatically if possible rather than manually adjusting each link. For now, I'd suggest leaving as is without the `(opens in a new window)`